### PR TITLE
Add BFL language definition and do syntax highlighting based on that

### PR DESCRIPTION
**Pull request by mertyildiran**
[BFL Syntax Reference](https://github.com/up9inc/basenine/wiki/BFL-Syntax-Reference)

### CODE REVIEW

**Comment by mertyildiran** [comment]
Blocked by https://github.com/uiwjs/react-textarea-code-editor/pull/85

File: ui-common/package-lock.json [hunk 1]
```diff
@@ -26566,6 +26529,22 @@
...
+      },
+      "peerDependencies": {
+        "@babel/runtime": ">=7.10.0",
+        "react": ">=16.9.0",
+        "react-dom": ">=16.9.0"
+      }
+    },
     "node_modules/react-toastify": {
       "version": "8.2.0",
       "resolved": "https://registry.npmjs.org/react-toastify/-/react-toastify-8.2.0.tgz",
```
Comment (Jamiejayz13) [inline ui-common/package-lock.json:26550]: React

File: ui-common/package-lock.json [hunk 2]
```diff
@@ -27397,11 +27376,11 @@
       }
     },
     "node_modules/registry-auth-token": {
-      "version": "4.2.1",
-      "resolved": "https://registry.npmjs.org/registry-auth-token/-/registry-auth-token-4.2.1.tgz",
-      "integrity": "sha512-6gkSb4U6aWJB4SF2ZvLb76yCBjcvufXBqvvEx1HbmKPkutswjW1xNVRY0+daljIYRbogN7O0etYSlbiaEQyMyw==",
```
Comment (Jamiejayz13) [inline ui-common/package-lock.json:27402]: Integrity

File: ui-common/package-lock.json [hunk 3]
```diff
@@ -27472,6 +27451,20 @@
         "url": "https://opencollective.com/unified"
       }
     },
+    "node_modules/rehype-prism-2": {
+      "version": "0.8.1",
+      "resolved": "https://registry.npmjs.org/rehype-prism-2/-/rehype-prism-2-0.8.1.tgz",
```
Comment (Jamiejayz13) [inline ui-common/package-lock.json:27456]: Resolved__

**Review by Jamiejayz13 — CHANGES_REQUESTED** [review]
Thank

**Comment by mertyildiran** [comment]
Closing since it's outdated.

**Pull request closed without merging**